docs/concepts/xapi_concepts.rst

```diff
@@ -4,47 +4,112 @@ xAPI
 ****
 
 Introduction
-###################
+############
 
-xAPI, also known as the Experience API or Tin Can API, is a specification for capturing
+xAPI, also known as the `Experience API`_ or Tin Can API, is a specification for capturing
 and tracking learning experiences in a wide range of contexts. It is an e-learning
 standard that allows organizations to gather data about learner activities and interactions
 across various platforms and devices. xAPI provides a flexible and comprehensive
 framework for collecting, storing, and analyzing learning data, enabling a deeper
 understanding of learners' experiences and performance.
 
-Components of xAPI
-###################
+Together, these components form the foundation of the xAPI ecosystem, enabling
+the tracking and analysis of learner activities and experiences beyond traditional
+learning management systems. The flexibility and interoperability of xAPI make it
+an ideal solution for organizations looking to gain deeper insights into their
+learners' performance and behavior.
 
-The xAPI specification consists of three key components:
+xAPI Schema
+###########
 
-.. _actor_concept:
+While the components of an :ref:`xapi_statement` are well specified, the actual detailed format or "schema" used for
+each type of xAPI event is up to us to decide, as users of xAPI.
 
-Actor
-~~~~~
-In xAPI, an actor represents an entity that interacts with the learning system.
-It can be a learner, instructor, system administrator, or any other agent
-involved in the learning process. Actors are identified using a unique identifier,
-such as an email address or username.
+See :ref:`xapi_transforms` for information on how the schemas are built.
+
+.. _xapi_statement:
+
+xAPI Statement
+##############
 
-Statement
-~~~~~~~~~
 A statement, also referred to as an "xAPI statement" or "verb-object statement,"
 is the core building block of xAPI. It captures a specific learning activity
 or experience in a structured format. A statement consists of three essential
 elements: the actor who performed the action, the verb that describes the action,
 and the object representing the target or learning experience.
 
+An xAPI statement can be expressed briefly as:
+
+  **Actor** **Verbed** an **Object** (within **Context**).
+
+ID
+~~
+
+Each xAPI event emitted by Open edX will contain a `Universally Unique Identifier`_ (UUID)
+which is generated from the event actor, timestamp, verb, and child event ID (if present).
+This ID is used to de-duplicate events, for example when re-processing old tracking logs.
+
+.. _actor_concept:
+
+Actor
+~~~~~
+In xAPI, an actor represents an entity that interacts with the learning system.
+It can be a learner, instructor, system administrator, or any other agent
+involved in the learning process.
+
+Actors in Aspects are uniquely identified using a platform-generated ``external_id`` which
```

Comment:
Might want to link in this doc: changing_actor_identifier somewhere here as it has some important specifics.

Reply:
Done: https://docsopenedxorg--114.org.readthedocs.build/projects/openedx-aspects/en/114/concepts/xapi_concepts.html#actor